CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_hubspot_source v0.12.0
+
+[PR #119](https://github.com/fivetran/dbt_hubspot_source/pull/119) includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+- The `created_at` field within the `stg_hubspot__contact` model has been renamed to `created_date` to be consistent with the source data. Additionally, this ensure there are no duplicate column errors when passing through all `property_*` columns which could potentially conflict with `property_created_at`.
+
+## Feature Updates
+- All `stg_hubspot__*_tmp` models have been updated to leverage the `dbt_utils.star()` macro. This ensures if the source dimension changes there is no potential for a mismatch in columns error that is commonly seen in Snowflake destinations.
```

Comment:
is this more of an under-the-hood thing?

Reply:
I guess it technically could be an under the hood thing. I can adjust the changelog to call switch that.